meeting-notes/2018/2018-08-13--golang-core-dev-team-weekly.md

```diff
@@ -0,0 +1,184 @@
+
+## Agenda
+
+- Ask everyone to put their name into the list of attendees
+- Round of updates
+  - What have you accomplished since the last Weekly?
+  - Were there any blockers? If so, which ones? Is it still blocked? Why?
+  - What is the next important thing you should focus on?
+- Ask for general questions. Could be things like:
+  - I'm stuck with something, I don't know who to ask. Who knows who to ask?
+  - Who can help me with xyz?
+- Plan this week
+  - Select issues to work on
+- Review remaining issues if there is time left
+
+
+## Notes
+
+### Week Update
+
+@stebalien
+ - Done:
+   - Reviewed a bunch of PRs
+   - Responded to/triaged a bunch of issues
+   - Fixed a few bugs
+   - Notable: update gogo protobuf
+ - Blocked:
+   - https://github.com/multiformats/multiaddr/pull/68 <- @daviddias P3
+   - https://github.com/multiformats/multiaddr/pull/68#issuecomment-401971665
+ - Next:
+   - More of the same
+     - CIDv1
+   - Finish the protocol negotiation spec
+
+
+@diasdavid
+ - Done:
+   - Kick off the Go Core Dev Weekly Sync
+   - Created https://github.com/ipfs/pm/blob/master/GOLANG_CORE_DEV_MGMT.md
+   - Updated the go-ipfs and go-libp2p waffle boards and created the go-ipld one 
+   - Go Core Dev Team Interviews (ongoing)
+   - Proposed Lead Maintainer Protocol up for discussion
+ - Blocked:
+   - none
+ - Next:
+   - Continue doing the Go Core Dev Team Interviews
+   - Encourage everyone to do some Waffle Grooming
+     - rough time scale: 30min/wk
+
+
+@marten-seemann (not attending)
+ - Done: 
+   - various QUIC improvements
+ - Blocked:
+   - not blocked by anything
+ - Next:
+   - re-evaluate TLS1.3 libraries for their use in QUIC
+   
+@kevina (not attending)
+  - Done:
+    - Add ability to retrieve blocks even if given using a different CID version (https://github.com/ipfs/go-ipfs/pull/5285)
+    - Builder interface for CID https://github.com/ipfs/go-cid/pull/53 & https://github.com/ipfs/go-ipfs/pull/5375
+    - Learned how to use gx-workspace and work with gx dependency managment in general
+    - Various work towards Base32 CidV1 (see Other notes section)
+  - Blocked:
+     - Various aspects of Base32 CidV1 need feedback (see Other notes section)
+  - Next:
+     - Continue Base32 CidV1 work
+     - Maybe add support for inlinling via the id-hash (https://github.com/ipfs/go-ipfs/pull/5281).
+    
+@lgierth (not attending)
+- done
+  - infra stuff, nothing go related
+- blocked
+  - nothing go related :)
+- next
+  - a few code reviews where i was summoned
+  
+@schomatis (not attending)
+ - Done:
+   - Reviewed a few PRs
+   - Studied part of the `gx`/`gx-go` code and workflow, how can we
+     make the package manager more user friendly?
+ - Next:
+   - Start submitting some PRs in `gx`/`gx-go` with documentation
+
+
+@bigs
+ - Done:
+   - Research into virtual network topologies w/ openvswitch/iptools
+     -> @daviddias introduce travis & jacobheun
```

Comment:
Let's do Github Style!

@travisperson @jacobheun, please meet @bigs. I'm sure you have crossed each other or even met at the IPFS Dev Meetings.

- @bigs is doing a lot of IPTB work for libp2p testing
- @travisperson is doing a lot of IPTB work to add support for js-ipfs daemons (Node.js) and js-ipfs browser nodes
- @jacobheun is looking into designing interop tests that can be used by go-libp2p, js-libp2p and rust-libp2p

It would be great to have you all sync soon to make sure that we leverage everyone's focus and don't get duplicated work done.